api/graphql/makeModels.js

```diff
@@ -198,10 +201,7 @@ export default function makeModels (userId, isAdmin) {
             sort: sortBy,
             topic
           })
-      },
-      filter: nonAdminFilter(relation => relation.query(q => {
```

Comment:
Removing this filter would give a user access to all the communities in the system through the top-level `community()` query. I think what you have to do here instead is keep the filter and do this:

```js
q => {
  q.where(function () {
    this.where('communities.id', 'in', myCommunityIds(userId))
    .orWhere('communities.id', 'in', myNetworkCommunityIds(userId))
  })
}
```